.travis.yml

```diff
@@ -32,7 +32,8 @@ matrix:
       env: CI_TARGET=clang-report SCAN_BUILD=scan-build
       compiler: clang
     - os: osx
-      osx_image: xcode10.1
+      # macOS 10.11. Keep this aligned with CMAKE_OSX_DEPLOYMENT_TARGET !
+      osx_image: xcode7.3
```

Comment:
Is this really needed? I think we can still use the latest Xcode and set `CMAKE_OSX_DEPLOYMENT_TARGET` to whatever version we want to support. Mind that it doesn't set an exact version but the _minimum version_ the executable is supposed to run on.

https://cmake.org/cmake/help/latest/variable/CMAKE_OSX_DEPLOYMENT_TARGET.html

Reply:
hmm, we can try that.